jetty-servlet/src/test/java/org/eclipse/jetty/servlet/AsyncContextTest.java

```diff
@@ -229,7 +229,7 @@ public void testDispatchAsyncContext() throws Exception
     @Test
     public void testDispatchAsyncContextEncodedUrl() throws Exception
     {
-        String request = "GET /ctx/test/hello%2fthere?dispatch=true HTTP/1.1\r\n" +
+        String request = "GET /ctx/test/hello%20there?dispatch=true HTTP/1.1\r\n" +
```

Comment:
Why was this test changed from `%2F` to `%20`? Seems like a big difference not relevant to this PR?

Reply:
The test is that we can dispatch to encoded paths correctly. We now treat %2f as an ambiguous path, so using %20 instead gives us the same test but without needing to switch compliance mode.